CMakeLists.txt

```diff
@@ -127,7 +127,7 @@ macro(get_boost)
 
 		  # First, try to find Boost without any hints (system Boost)
 		  if(NOT VITA3K_FORCE_CUSTOM_BOOST)
-		      find_package(Boost 1.81 COMPONENTS ${BOOST_MODULES_TO_FIND} QUIET)
+		      find_package(Boost 1.83 COMPONENTS ${BOOST_MODULES_TO_FIND} QUIET)
```

Comment:
Why 1.83 is required instead of 1.81?

Reply:
i just for increment minimum requiement
but for now i try fix one things...

Reply:
I do agree with scribam, the minimum version should only be increased if it is required.

Reply:
Ah, i have thinks is good with see all workflow using 1.83 🥲

Reply:
I have revert it is ready